### PR TITLE
[TGCS] Update EvaluateHopper

### PR DESCRIPTION
**Pull request by carmichaelong**
This PR contains a few small changes before using it in the interactive GUI setup. There are 2 updates to the EvaluateHopper function:
1) Uses the osimTableToStruct utility, and removes the utility in the folder.
2) Removes metabolics and device penalties for now (we can revisit scoring later).

This also merges changes to configureOpenSim that is waiting in another PR to master.

@chrisdembia @nickbianco

### CODE REVIEW

File: Bindings/Java/Matlab/CMakeLists.txt [hunk 1]
```diff
@@ -8,38 +8,14 @@ endif()
 # Install.
 # --------
 install(DIRECTORY examples/ DESTINATION "${OPENSIM_INSTALL_MATLABEXDIR}")
-install(DIRECTORY Hopper_Device DESTINATION "${OPENSIM_INSTALL_MATLABEXDIR}")
+install(DIRECTORY Utilities DESTINATION "${OPENSIM_INSTALL_MATLABEXDIR}")
```
Comment (chrisdembia) [inline Bindings/Java/Matlab/CMakeLists.txt]: I think you still want to install the Hopper_Device folder.

**Comment by chrisdembia** [comment]
@nickbianco FYI: After this is merged, you'll have to add the Utilities directory (in the installation) to your MATLAB path. The configureOpenSim script (must be run from the install directory) does this for you.

**Comment by carmichaelong** [comment]
> Did you run RunHopper_answers and RunHopperWithDevice_answers to ensure they still work as intended (and that they print the table to the MATLAB command window)?

Oops. Was just running `BuildHopper` and then testing with `EvaluateHopper`. Thanks for catching that I hadn't touched the answers. Pushed some changes and tested, and they do work with the table utility. Note that there were a couple of changes to the table utility itself pushed here too.

**Comment by chrisdembia** [comment]
Okay awesome.

**Comment by chrisdembia** [comment]
@carmichaelong I deleted the branch.

**Pull request merged**